jsonnet/telemeter/client/kubernetes.libsonnet

```
@@ -92,7 +92,7 @@ local securePort = 8443;
       local containerVolumeMount = container.volumeMountsType;
       local containerEnv = container.envType;
 
-      local podLabels = { 'k8s-app': 'telemeter-client' };
+      local podLabels = $._config.commonLabels + { 'k8s-app': 'telemeter-client' };
```

Now that we are using the recommended Kubernetes labels, we can remove k8s-app.
```suggestion
      local podLabels = $._config.commonLabels;
```

---

👍